README.md

```diff
@@ -376,7 +376,7 @@ Type: String
 
 Default: `eni`
 
-Specifies the veth prefix used to generate the host-side veth device name for the CNI. The prefix can be at most 4 characters long.
+Specifies the veth prefix used to generate the host-side veth device name for the CNI. The prefix can be at most 4 characters long. The prefixes `eth` or `vlan` or `lo` are reserved and cannot be specified. We recommend using exclusive prefix for the host side veth device names.
```

Comment:
Are they really reserved though? We do create an eth0/1 in container ns. If the idea is to encourage users to avoid specifying some common linux interface prefixes, maybe we can have a generic statement here suggesting to avoid them(?). Because we probably can include more prefixes beyond the ones specified here (tun, tap, br etc)....

Reply:
These are not reserved by linux, these are reserved only by the CNI plugin.

Reply:
Might help to call that out specifically. Also, we basically don’t want any other interfaces on the instance to share the same prefix as host veth interfaces and I think we should specify that in the above description so that customers know the prefixes to avoid in addition to the above three we are validating.

Reply:
updated readme.